tools/PI/DevHome.PI/Helpers/ExternalToolsHelper.cs

```diff
@@ -58,6 +60,8 @@ public ObservableCollection<ExternalTool> AllExternalTools
     // The bar shows only the pinned tools.
     public ReadOnlyObservableCollection<ExternalTool> FilteredExternalTools { get; private set; }
 
+    internal static int ToolsCollectionVersion { get; private set; } = 2;

```

Comment:
Version information inside the code?  Maybe this should be moved to the .csproj file.  Also, why a version?

Reply:
This is the file format version, which maps to the data that we're serializing/deserializing.